src/MessageTile/MessageTile.jsx

```diff
@@ -9,7 +9,10 @@ const styles = theme => ({
   box: {
     borderRadius: theme.spacing(0.5),
     marginBottom: theme.spacing(2),
```

Comment:
I'm not following 100%.

Should the `marginBottom` spacing be changed to `theme.spacing(1)` (i.e. 8px)? If the bottom padding is 2 units, and the bottom margin is 1 unit, then that adds up to 3 units (i.e. 24px).

Reply:
`MessageTile`'s margin is spacing _outside_ the containing box, so it's for spacing between it and other components. 

I do believe, however, that I've got my units mixed up. The code is right, but not the title. Apologies for the confusion.

The padding on the `MessageTile` is 24px 16px 16px 16px. We're wanting spacing of 24px 16px overall.

With the `MessageTile`'s padding-bottom of 16px, and the `MessageTileButton`/`MessageTileFooter` having a margin-bottom of 8px, there's the 24px we need to get a _spacing_ of 24px 16px.

Reply:
I've amended the title.